web_src/js/components/ViewFileTreeItem.vue

```diff
@@ -16,14 +16,19 @@ const props = defineProps<{
   item: Item,
   navigateViewContent:(treePath: string) => void,
   loadChildren:(treePath: string, subPath?: string) => Promise<Item[]>,
+  getWebUrl:(treePath: string) => string,
   selectedItem?: string,
 }>();
 
 const isLoading = ref(false);
 const children = ref(props.item.children);
 const collapsed = ref(!props.item.children);
 
-const doLoadChildren = async () => {
+const doLoadChildren = async (e?: MouseEvent) => {
```

Comment:
```suggestion
const doLoadChildren = async (e: MouseEvent | null) => {
```

I guess I would prefer if you pass explicit null here to indicate the absence of a event.